packages/wonder-blocks-toolbar/components/toolbar.md

```diff
@@ -0,0 +1,105 @@
+### Small toolbar example:
+
+```js
+const Button = require("@khanacademy/wonder-blocks-button").default;
+const smallButton = <Button size="small">Small button</Button>;
+const secondaryButton = <Button size="small" kind="secondary">Secondary button</Button>;
+
+<Toolbar size="small" rightContent={[smallButton, secondaryButton]} />
+```
+
+### Toolbar with title
+
+```js
+// TODO(scottgrant): Use wonder-blocks-icon and wonder-blocks-icon-button
```

Comment:
This is in a separate PR.

Reply:
Sorry, are you saying to remove the TODO? I totally agree that it should be in a separate PR, but I thought a TODO made sense. Maybe not though! What's your preference here?

Reply:
I was agreeing with the TODO and saying that this doesn't need to be fixed in this PR.